modules/add-to-device-calendar/lib.js

```diff
@@ -1,7 +1,6 @@
 // @flow
 
 import type {EventType} from '@frogpond/event-type'
-import {notify} from '@frogpond/analytics'
```

Comment:
Thinking into the app logic… I see a lot of notify/breadcrumb removals. Do we want to keep those? Does sentry automatically do them?

Reply:
Sentry "should" handle crashes and the like, I believe. For sure, it includes the necessary beef to handle unhandled promise rejections, but beyond that I am not sure if it will e.g. catch uncaught errors.

Reply:
Addendum: `notify` was mostly used to explicitly send out an event or error message, and we will need to manually use the Sentry equivalent of that.